packages/preact/src/index.ts

```diff
@@ -465,6 +472,29 @@ export function useSignalEffect(
 	}, []);
 }
 
+function JSXBind({ cb }: { cb: () => unknown }) {
```

Comment:
What do you think about the following, I am also removing the eager destructuring to save some bytes

```suggestion
type JSXBindable =
  | string
  | number
  | boolean
  | null
  | undefined
  | VNode<any>
  | ComponentChildren;

type JSXBindCallback<T extends JSXBindable = JSXBindable> = () => T;

function JSXBind(props: { cb: JSXBindCallback }) {
```

Reply:
I managed to remove the `JSXBind` component entirely 👍